src/options-gsettings.vala

```diff
@@ -45,9 +43,13 @@ public class IndicatorSound.OptionsGSettings : Options
         private void init_max_volume() {
                 _settings.changed[NORMAL_dB_KEY].connect(() => update_max_volume());
                 _settings.changed[AMP_dB_KEY].connect(() => update_max_volume());
-#if LOMIRI_FEATURES_ENABLED
```

Comment:
Shouldn't we combine both? #if LOMIRI_FEATURES_ENABLED _and_ AyatanaCommon.utils_is_lomiri() ???

Reply:
We could, but I think we should make these code portions fail on Lomiri in case LOMIRI_FEATURES_ENABLED is omitted. If you disagree, I'll put back the ifdef.

Reply:
Problem with failing indicators is, you don't get any kind of error message or anything. Furthermore, the build options are distro maintainer tasks, so the user should not suffer if the distro maintainer omitted LOMIRI_FEATURES_ENABLED.

Please bring back the ifdefs. Thanks.

Reply:
Done.